### PR TITLE
fix: improve error message output in integration tests

### PR DESCRIPTION
**Pull request by gfanton**
This PR improves error output in integration tests by using more verbose format.

`gnokey` will now display the full stack trace log, including the underlying error, when an error occurs, instead of only presenting the top-level error.

This enhancement will assist in better debugging what's happening without the need to delve directly into gnoland logs.

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

File: tm2/pkg/crypto/keys/client/addpkg.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ func signAndBroadcast(
 
 	signedTx, err := SignHandler(sopts)
 	if err != nil {
-		return errors.Wrap(err, "sign tx")
+		return fmt.Errorf("unable to sign tx: %w", err)
```
Comment (moul) [inline tm2/pkg/crypto/keys/client/addpkg.go:198]: It looks strange that we need to do this. Can’t we improve the display logic of the errors package?

**Comment by moul** [comment]
I prefer the new output 👍

Can you check how much we could improve the package/errors instead of avoiding using it?

**Comment by gfanton** [comment]
@moul sure ! Let me check

**Comment by gfanton** [comment]
I've reverted my changes and will simply use "%+v" instead of "%v" for the integration tests. It took me a while to notice that we are using a custom format for our errors. For now, this change is sufficient as my main goal was to improve the error output of the gnokey command within the integration tests.

That said, I believe there is room for improvement in error handling, as we should not need to use this kind of format to obtain the source error. Do you think we can update our errors package to use a more recent format, wrapping `%w`, to improve error readability? (in a separate pull request)

**Review by deelawn — APPROVED** [review]

**Review by thehowl — APPROVED** [review]

**Pull request merged**